twilight-gateway/src/shard.rs

```diff
@@ -280,9 +285,10 @@ struct MinimalReady {
 /// # Ok(()) }
 /// ```
 ///
-/// [`queue`]: crate::queue
 /// [docs:shards]: https://discord.com/developers/docs/topics/gateway#sharding
 /// [gateway commands]: Shard::command
+/// [`stream`]: crate::stream
+/// [`queue`]: crate::queue
```

Comment:
just a footnote: the sorting being changed here is probably due to my inconsistent use of numeric sorting in vim